src/plugins/intel_gpu/tests/test_cases/bucketize_gpu_test.cpp

```diff
@@ -41,25 +46,34 @@ struct bucketize_test : testing::TestWithParam<bucketize_test_params<I, B, O>> {
 
         const layout buckets_layout(type_to_data_type<B>::value,
                                     format::bfyx,
-                                    tensor(format::bfyx, {1, 1, 1, static_cast<int>(p.buckets_values.size())}));
+                                    tensor(format::bfyx, {static_cast<int>(p.buckets_values.size()), 1, 1, 1}));
         auto buckets = engine.allocate_memory(buckets_layout);
         set_values(buckets, p.buckets_values);
 
         topology topology;
         topology.add(input_layout("input", input->get_layout()));
         topology.add(input_layout("buckets", buckets->get_layout()));
-        topology.add(bucketize("bucketize_right_bound", {"input", "buckets"}, type_to_data_type<O>::value, true));
-        topology.add(bucketize("bucketize_left_bound", {"input", "buckets"}, type_to_data_type<O>::value, false));
+        topology.add(reorder("reordered_input", "input", fmt, type_to_data_type<I>::value));
+        topology.add(reorder("reordered_buckets", "buckets", fmt, type_to_data_type<B>::value));
+
+        topology.add(
+            bucketize("bucketize_right_bound", {"reordered_input", "buckets"}, type_to_data_type<O>::value, true));
+        topology.add(
+            bucketize("bucketize_left_bound", {"reordered_input", "buckets"}, type_to_data_type<O>::value, false));
+        topology.add(
+            reorder("plane_bucketize_right_bound", "bucketize_right_bound", format::bfyx, type_to_data_type<O>::value));
+        topology.add(
+            reorder("plane_bucketize_left_bound", "bucketize_left_bound", format::bfyx, type_to_data_type<O>::value));
 
         network network(engine, topology);
         network.set_input_data("input", input);
         network.set_input_data("buckets", buckets);
         const auto outputs = network.execute();
 
-        EXPECT_EQ(outputs.size(), size_t(2));
+        //   EXPECT_EQ(outputs.size(), size_t(2));
```

Comment:
Maybe delete or uncomment this line?

Reply:
will remove, as it doesn't make big sense